Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -76,15 +81,14 @@ private double calcCharityTrendingScore(Charity charity) {
         if (hasCharityNavRating) {
             charityNavRating = calcCharityNavRating(charity);
         }
-        //userRating = GET request to db for charity star rating;
         double userRating = charity.getUserRating();
         double avgReview;
         if (hasCharityNavRating) {
             avgReview = userRatingWeight * userRating + scaledCharityNavWeight * charityNavRating;
         } else {
             avgReview = userRating;
         }
-        //Collection<Tag> tags = GET request to db for charity tags;
+        //getCategories?
```

Comment:
do you want to elaborate more in the comment? And remove "?"

Reply:
done